src/NUnitConsole/nunit3-console/Program.cs

```diff
@@ -64,6 +65,20 @@ public static int Main(string[] args)
                 return ConsoleRunner.INVALID_ARG;
             }
 
+            if (!string.IsNullOrEmpty(Options.ConsoleEncoding))
+            {
+                try
+                {
+                    Console.OutputEncoding = Encoding.GetEncoding(Options.ConsoleEncoding);
+                }
+                catch (Exception error)
+                {
+                    WriteHeader();
+                    OutWriter.WriteLine(ColorStyle.Error, string.Format("Unsupported Encoding! {0}", error.Message));
```

Comment:
Please, nitpick all you want. I put this together rather quickly, so I'm expecting formatting changes.